extra/utils/permision.js

```diff
@@ -0,0 +1,70 @@
+let permissions= {
+    'getUsers': {
+        'all': ['head-trainer'],
+        'read': ['trainee', 'trainer'],
+        'write': ['trainer'],
+        'delete': [],
+    }
+}
+
+function hasPermission(moduleName,role,permissionType)
+{
+   
+    if(permissions[moduleName]!== undefined)
+    {
+        console.log("Module Name ="+moduleName)
+        if(permissions[moduleName][permissionType]!==undefined)
+        {
+            console.log("Permission Type ="+permissionType)
+            if(permissions[moduleName][permissionType]!=='all')
```

Comment:
You are tring to compare array with string value. There is no need to do that since you have already checked for permissions[moduleName][permissionType]!==undefined